src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+$text-color: #060B35;
+$secondary-text-color:  #616070;
+$button-color: #00ACDC;
+$main-page-color: #ffffff;
+$navigation-color: #060b35;
```

Comment:
The color value for `$navigation-color` is `#060b35`, which is the same as `$text-color` but with different capitalization. Consider using consistent capitalization for hex color codes to maintain uniformity.